packages/core/admin/admin/src/components/AuthenticatedApp/utils/api.js

```diff
@@ -38,7 +39,7 @@ const fetchStrapiLatestRelease = async (toggleNotification) => {
 
 const fetchAppInfo = async () => {
   try {
-    const { data, headers } = await axiosInstance.get('/admin/information');
+    const { data, headers } = await get('/admin/information');
```

Comment:
Could you please make sure all calls prefixed with `/admin` take `process.env.STRAPI_ADMIN_BACKEND_URL` into account?

Reply:
We can't use just the env var, we should be using the internal API method to fetch the admin config, eg in both these examples your env var wouldn't work

![image](https://user-images.githubusercontent.com/8593673/209177850-1e1f1167-339b-43a3-aaa7-0faf3564037a.png)

Reply:
Thanks guys, this activity will be part of another ticket just to focus on the axiosInstance replacement on this one. This issue https://github.com/strapi/strapi/issues/13889 will be fixed in another ticket with some activities in the BE and in the FE if needed